bindings/java/src/junit/com/apple/foundationdb/tuple/ArrayUtilTests.java

```diff
@@ -248,61 +252,61 @@ public void testSplit() {
 		}
 	}
 
-	/**
-	 * Test method for {@link ByteArrayUtil#bisectLeft(java.math.BigInteger[], java.math.BigInteger)}.
-	 */
-	@Test
-	public void testBisectLeft() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#compareUnsigned(byte[], byte[])}.
-	 */
-	@Test
-	public void testCompare() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#findNext(byte[], byte, int)}.
-	 */
-	@Test
-	public void testFindNext() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#findTerminator(byte[], byte, byte, int)}.
-	 */
-	@Test
-	public void testFindTerminator() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#copyOfRange(byte[], int, int)}.
-	 */
-	@Test
-	public void testCopyOfRange() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#strinc(byte[])}.
-	 */
-	@Test
-	public void testStrinc() {
-		fail("Not yet implemented");
-	}
-
-	/**
-	 * Test method for {@link ByteArrayUtil#printable(byte[])}.
-	 */
-	@Test
-	public void testPrintable() {
-		fail("Not yet implemented");
-	}
+//	/**
+//	 * Test method for {@link ByteArrayUtil#bisectLeft(java.math.BigInteger[], java.math.BigInteger)}.
+//	 */
+//	@Test
```

Comment:
We could add an `@Ignore` annotation instead of commenting them out, which has the advantage of the compiler still type checking this code

Reply:
Oh - the unit tests didn't run properly anyways. I now fixed this and I also made this change (was not aware of the `@Ignore` annotation)